TKeazirian.HTTPServer/Program.cs

```diff
@@ -23,18 +23,31 @@ private static void StartListening()
 
             Console.WriteLine("Waiting for a connection...");
             var handler = listener.Accept();
-            IncomingData = null;
+            request = null;
 
             byte[] bytes = new byte[1024];
 
             int bytesReceived = handler.Receive(bytes);
-            IncomingData += Encoding.ASCII.GetString(bytes, 0, bytesReceived);
 
-            Console.WriteLine($"Text received: {IncomingData}");
+            request += Encoding.ASCII.GetString(bytes, 0, bytesReceived);
```

Comment:
Looks like this is never actually reassigned. Can this just be something like
`string request = Encoding.ASCII...`

Reply:
good catch - fixed!